frontend/src/modules/news/page/BrowseNewsPage/index.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import Home from '.';
+
+describe('browseNews page', () => {
+  it('browseNews page should be defined', () => {
+    const wrap = shallow(<Home />);
+    expect(wrap.exists()).toBe(true);
```

Comment:
test should able to detect a bad behavior/import, please check this gist to let you have an idea for testing.

https://github.com/yee2542/FluentSearch-FE/tree/master/src/modules/photos/components/SelectBadgeIcon
https://github.com/yee2542/FluentSearch-FE/tree/master/src/modules/photos/components/ThumbnailPhoto